tests/e2e-leg-11/client-proxy/27-verify-client-proxy-config.yaml

```diff
@@ -0,0 +1,59 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: script-client-proxy-config
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    set -o errexit
+    set -o xtrace
+    
+    # get the proxy pod name
+    PROXY_PRI1=$(kubectl get pod -l pod-template-hash,vertica.com/subcluster-svc=pri1 --no-headers -o custom-columns=:.metadata.name)
+    
+    # get the content of the config.yaml
+    CONFIG=$(kubectl exec -i $PROXY_PRI1 -- sh -c "cat /config/config.yaml")
+    
+    for POD_NAME in v-client-proxy-pri1-0 v-client-proxy-pri1-1 v-client-proxy-pri1-2; do
+      RESULT=$(echo "$CONFIG" | grep -Pzo "$POD_NAME:5433" | tr -d '\0')
+      if [ "$POD_NAME:5433" != "$RESULT" ]; then
+        echo "Assertion failed: expect \"$POD_NAME:5433\" to be defined in config.yaml, got \"$RESULT\""
+        exit 1
+      fi
+    done
```

Comment:
Run a vsql command through one of the vertica pod to verify we can connect to the proxy, and verify the hostname(we could query current_session) is the proxy hostname.

Reply:
This will be a task for next service-label jira.